packages/server/lib/util/lazyDecorator.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * A lazy decorator means the value is lazily evaluated once and
```

Comment:
I stole this from the services repo. Is there an easy way to share code between services and cypress?

Reply:
As of right now, no. I think some copy paste is probably fine for now, once we have an idea of what we'd like to share, maybe we can create a shared package.